spec/checkPullRequestTemplateSpec.js

```diff
@@ -0,0 +1,471 @@
+      payloadData.payload.pull_request.maintainer_can_modify = false;
+      await robot.receive(payloadData);
+    });
+
+    it('should check the template', () => {
+      expect(checkPullRequestTemplateModule.checkTemplate).toHaveBeenCalled();
+    });
+
+    it('should ping PR author', () => {
+      expect(github.issues.createComment).toHaveBeenCalled();
+      expect(github.issues.createComment).toHaveBeenCalledWith({
+        issue_number: payloadData.payload.pull_request.number,
+        repo: payloadData.payload.repository.name,
+        owner: payloadData.payload.repository.owner.login,
+        body:
+          'Hi @' +
+          payloadData.payload.pull_request.user.login +
+          ', the allow edits from maintainers checklist needs to ' +
+          'be ticked so that maintainers can rerun failed tests. Endeavour ' +
+          'to add this by ticking on the check box. Thanks!',
+      });
+    });
+
+    it('should assign PR author', () => {
+      expect(github.issues.addAssignees).toHaveBeenCalled();
+      expect(github.issues.addAssignees).toHaveBeenCalledWith({
+        issue_number: payloadData.payload.pull_request.number,
+        repo: payloadData.payload.repository.name,
+        owner: payloadData.payload.repository.owner.login,
+        assignees: [payloadData.payload.pull_request.user.login],
+      });
+    });
+  });
+
+  describe('when pull request contains filled checklist without description', () => {
+    beforeEach(async () => {
+      payloadData.payload.pull_request.body = bodyWithValidChecklistButNoDescription;
+      payloadData.payload.pull_request.maintainer_can_modify = true;
+      await robot.receive(payloadData);
+    });
+
+    it('should check the template', () => {
+      expect(checkPullRequestTemplateModule.checkTemplate).toHaveBeenCalled();
+    });
+
+    it('should ping PR author', () => {
+      expect(github.issues.createComment).toHaveBeenCalled();
+      expect(github.issues.createComment).toHaveBeenCalledWith({
+        issue_number: payloadData.payload.pull_request.number,
+        repo: payloadData.payload.repository.name,
+        owner: payloadData.payload.repository.owner.login,
+        body:
+          'Hi @' +
+          payloadData.payload.pull_request.user.login +
+          ', the body of this PR is missing the required description, ' +
+          'please update the body with a description of what this PR does. ' +
+          'Thanks!',
+      });
+    });
+
+    it('should assign PR author', () => {
+      expect(github.issues.addAssignees).toHaveBeenCalled();
+      expect(github.issues.addAssignees).toHaveBeenCalledWith({
+        issue_number: payloadData.payload.pull_request.number,
+        repo: payloadData.payload.repository.name,
+        owner: payloadData.payload.repository.owner.login,
+        assignees: [payloadData.payload.pull_request.user.login],
+      });
+    });
+  });
+
+  describe('when pull request contains completely valid body', () => {
```

Comment:
Remove completely.

Reply:
Removing this will reduce code coverage.

Reply:
Sorry, I meant remove the word completely.

Reply:
Oh okay.
Done.